MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/SatelliteLandSelectActivity.java

```diff
@@ -77,34 +79,39 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   @Override
-  public void onMapReady(MapboxMap mapboxMap) {
+  public void onMapReady(@NonNull final MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
 
-    mapboxMap.animateCamera(
-      CameraUpdateFactory.newCameraPosition(new CameraPosition.Builder()
-        .zoom(16)
-        .build()), 4000);
-
-    fillLayerPointList = new ArrayList<>();
-    circleLayerFeatureList = new ArrayList<>();
-    lineLayerPointList = new ArrayList<>();
-
-    // Add sources to the map
-    initCircleSource();
-    initLineSource();
-    initFillSource();
-
-    this.circleSource = mapboxMap.getSourceAs(CIRCLE_SOURCE_ID);
-    this.fillSource = mapboxMap.getSourceAs(FILL_SOURCE_ID);
-    this.lineSource = mapboxMap.getSourceAs(LINE_SOURCE_ID);
-
-    // Add layers to the map
-    initCircleLayer();
-    initLineLayer();
-    initFillLayer();
-
-    initFloatingActionButtonClickListeners();
-    Toast.makeText(this, R.string.trace_instruction, Toast.LENGTH_LONG).show();
+    mapboxMap.setStyle(Style.SATELLITE, new Style.OnStyleLoaded() {
+      @Override
+      public void onStyleLoaded(@NonNull Style style) {
+        mapboxMap.animateCamera(
+          CameraUpdateFactory.newCameraPosition(new CameraPosition.Builder()
+            .zoom(16)
+            .build()), 4000);
+
+        fillLayerPointList = new ArrayList<>();
+        circleLayerFeatureList = new ArrayList<>();
+        lineLayerPointList = new ArrayList<>();
```

Comment:
can be initialized at field creation and made final

Reply:
Fixed